src/Compilers/CSharp/Portable/Symbols/MemberSignatureComparer.cs

```diff
@@ -218,7 +219,7 @@ internal sealed class MemberSignatureComparer : IEqualityComparer<Symbol>
             considerReturnType: false,
             considerTypeConstraints: false,
             considerCallingConvention: false, //ignore static-ness
-            considerRefKindDifferences: true,
+            refKindCompareMode: RefKindCompareMode.ConsiderDifferences | RefKindCompareMode.AllowRefReadonlyVsInMismatch,
```

Comment:
Same question for `CSharpAccessorOverrideComparer` (I suspect we may not be able to hit that one though)